src/promptflow-core/promptflow/_utils/inputs_mapping_utils.py

```diff
@@ -59,6 +59,14 @@ def apply_inputs_mapping(
         match = re.search(r"^\${([^{}]+)}$", map_value)
         if match is not None:
             pattern = match.group(1)
+            if pattern == CONVERSATION_HISTORY_EXPRESSION_KEY:
```

Comment:
I think this feature is only for chat group, may not be a good idea to put it in general inputs mapping logic?

Reply:
I move conversation history init logic in orchestrator